packages/requisitions/src/SupplierRequisition/DetailView/Toolbar.tsx

```diff
@@ -61,7 +61,7 @@ export const Toolbar: FC<ToolbarProps> = ({ draft }) => {
                     disabled={!isRequisitionEditable(draft)}
                     value={draft.otherParty}
                     onChange={name => {
-                      draft.update?.('otherParty', name);
+                      draft.updateOtherParty(name);
```

Comment:
the toolbar also needs the MOS / Stock Target field and the view filters. and, now that I look at it.. no option to delete lines

Reply:
Ah yes another question area. I think it might be good if I could meet with you or someone from the ui team and go through supplier requisitions to make sure I understand it all.

I wasn't sure if the option not to delete lines was for program requisitions or general or something, maybe

Mobile also has some other bulk actions (Create an auto order, set requested to forecast) that I thought might be nice to haves?

Reply:
yeah - I wasn't sure about requisitions, the initial discussions were only between Richard & Manu. Just noticed that there's no delete on the zeplin designs, though I suspect that's an oversight. As I understand it, only program reqs would be restricted in that way.

Would be great to bring the other actions through from mobile!